client/src/components/stadiumProfile/EditInput.tsx

```diff
@@ -0,0 +1,79 @@
+import { Edit } from '@mui/icons-material';
+import { Button, IconButton, InputAdornment } from '@mui/material';
+import { ReactElement, useState } from 'react';
+import { EditInputs } from './StadiumProfile.styled';
+
+interface Props {
+  lastValue: string | number;
+  multiline: boolean;
+}
+
+const EditInput: React.FC<Props> = ({
+  lastValue,
+  multiline,
+}: Props): ReactElement => {
+  const [editMode, setEditMode] = useState(false);
+  const [mouseOver, setMouseOver] = useState(false);
+  const [value, setValue] = useState(lastValue);
+  const [hov, setHove] = useState(false);
+  const [EditAble, setEditAble] = useState(true);
```

Comment:
this is added as a state and it is never updated. I think it will be updated when you can access the token to make sure that this is the user for this page, is it? or you miss this one.

Reply:
yes exactly